### PR TITLE
Tree preOrder traversal code added and README updated

### PR DESCRIPTION
**Pull request by pradeep98**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented preorder traversal for a tree using both recursion and iterative approach in PYTHON. Issue opened by @Abhijit2505

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
[Traversals in Binary Tree #415](https://github.com/TesseractCoding/NeoAlgo/issues/415)

### CODE REVIEW

**Review by Abhijit2505 — CHANGES_REQUESTED** [review]
Please remove the comments from line 23 to the end of the code. Try to add one line of explanatory comment on the top of any module, that makes the code look better as well as clean.

**Comment by pradeep98** [comment]
> Please remove the comments from line 23 to the end of the code. Try to add one line of explanatory comment on the top of any module, that makes the code look better as well as clean.

Comments modified. Please review the changes.

File: Python/ds/Tree_preOrder_traversal.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+preOrdertree.root.left  = node(4)
+preOrdertree.root.right = node(8)
+
+preOrdertree.root.left.left   = node(16)
+preOrdertree.root.left.right  = node(32)
+preOrdertree.root.right.left  = node(64)
+preOrdertree.root.right.right = node(128)
+
+'''
```
Comment (vatsalkesarwani12) [inline Python/ds/Tree_preOrder_traversal.py:64]: shift this comment to the bottom of the code

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
Make a change to the comments

File: Python/ds/Tree_preOrder_traversal.py (hunk 1 above)
Comment (pradeep98) [reply Python/ds/Tree_preOrder_traversal.py:64]: Comment shifted. Please review.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @pradeep98 Good work!
The code is working fine. Can you please make this code to take input from user as harcoded value isn't a good approach for CP

**Review by vatsalkesarwani12 — APPROVED** [review]
make dynamic input and the code works fine

**Comment by pradeep98** [comment]
> Hey @pradeep98 Good work!
> The code is working fine. Can you please make this code to take input from user as harcoded value isn't a good approach for CP

Dynamic input added!

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :)

**Comment by pradeep98** [comment]
@Abhijit2505 please review the changes.

**Review by Abhijit2505 — APPROVED** [review]

**Pull request merged**